src/bitcoin-cli.cpp

```diff
@@ -494,9 +494,6 @@ static int CommandLineRPC(int argc, char *argv[])
             }
         } while (fWait);
     }
-    catch (const boost::thread_interrupted&) {
-        throw;
-    }
     catch (const std::exception& e) {
```

Comment:
The catch above is to give special treatment to `boost::thread_interrupted`, relative to the `std::exception` catch that follows. It's a way of excluding these from the `std::exception` treatment.